src/mongo/Config.class.php

```diff
@@ -121,6 +121,7 @@ class Config
 
     const VALIDATE_MIN = 'MIN';
     const VALIDATE_MAX = 'MAX';
+    const CONNECTION_RETRIES = 30;
```

Comment:
so the longest we want it to retry for is 30 seconds given we have a 30 second cut off in cloudfront for https tenancies. which makes sense.